orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationResponse.java

```diff
@@ -0,0 +1,33 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.orchestration.client.model.CompletionPostResponse;
+import com.sap.ai.sdk.orchestration.client.model.LLMModuleResultSynchronous;
+import javax.annotation.Nonnull;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/** Orchestration chat completion output. */
+@RequiredArgsConstructor
```

Comment:
```suggestion
@RequiredArgsConstructor(access = PACKAGE)
```

Can be made package private, right?

Reply:
Also, add equals, hash code and tostring 

Maybe just use @value or consider a record